tests/testapps/CTCatalog/CTCatalog/Tests/CTCFrameTestViewController.mm

```diff
@@ -27,6 +26,11 @@ @interface CTFrameLineTestView : UIView {
 @implementation CTFrameLineTestView {
 }
 
+- (void)setLineRef:(CTLineRef)lineRef {
+    _lineRef = lineRef;
```

Comment:
er, CFRelease it.

---
In reply to: [79957519](https://github.com/Microsoft/WinObjC/pull/1010#discussion_r79957519) [](ancestors = 79957519)